### ISSUE TITLE
[Feature Request][3.6.6] Make VBtn & VChip outlined variant have the same visual width as non-outlined variants

### ISSUE THREAD

**Issue by frederikheld** [vuetifyjs/vuetify#19836]
### Environment
**Vuetify Version:** 3.6.6
**Vue Version:** 3.4.27
**Browsers:** Firefox 126.0
**OS:** Ubuntu undefined

### Steps to reproduce
In my case: use the v-chip as toggle-item that shows the two states as flat and outlined variants. You will see how the width changes which can sometimes unexpectedly wrap the line if multiple chips are displayed inline.

### Expected Behavior
Chip should have same width regardless of variant.

### Actual Behavior
Chip changes width depending on variant

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNp1j70OwjAMhF/FykybAalDCZV4BkbKkCahjZomUf6kCvHumHZhgM2n830+354kBkEv3tclK9ISltTiDU+q6y0Ak7pATKtR554MXMxjcNnKSjjjQgtByRNIHTGwtqCt0VZVg3Fi7smWR0KpxKQ9FB40twkxD6SjfXWLSpO2I6P7SseGAPRvzOX0wcuf0a0rxbI4Mfr1AsoogvYJokoZb9BdokVeB5uNOZBj3dQNub8BUHdbvA==)

**Comment by frederikheld** [comment]
I was able to "fix" this by conditionally adding a span around the v-chip text that has a margin left and right of 0.85em each. But I don't know if this is the right unit to keep it consistent and if this solution would work in other contexts as well.

**Comment by mirocklez** [comment]
The same applies for the button, as well - the outlined variant increases the width by 2px.

**Comment by frederikheld** [comment]
Okay, well. It's just too obvious that the border would be a pixel value XD I replaced 0.085em with 1px on each side and can confirm the fix.

**Comment by MatthewAry** [comment]
I'd consider this to be a pretty typical issue with anything that has a border applied to it, since the border adds to the dimensions of the element for it's purposes. Ideally, for design purposes, they would be identical in width. This is not really a bug though as nothing here is actually broken.

**Comment by MetRonnie** [comment]
@J-Sek I don't think this is a duplicate of #9087 - that is about inconsistencies between different border thicknesses, whereas this is about `variant=outlined` components having different widths to other variants

**Comment by Estheisia** [comment]
Maybe adding as a default on all variant : `border: 1px transparent` would provide the expected behavior, `outlined` would override with its own border while not having those 2 extra pixels ?

---

EDIT : doing the border transparent like I did is conflicting with the disable state : the btn got a plain-colored border instead of the disabled-tonal one
Maybe it would be easier to "just" add a colored border with non-0 width for not disabled ?
```
.v-btn {
  border-color: rgba(var(--v-border-color), var(--v-border-opacity));
  border-style: solid;
  border-width: 0;
}
```

[New example](https://play.vuetifyjs.com/#eNrdV1tv2zYU/iuc+iAZsCRnSVHMi9MUQ4sWGIZiDfYS5YGWjmx2FKmRlJMg8H/f4UWylBhdig4o0ABCpHP9zoU8x9cPEWtaqUza0Db7rKWIltFDIQgpAkMX0ZI4iqXtOrDfRbQ1ptXLPC8rgWoVcLZTmQCTi7bJL1EsV50wrIG0ks3lafYyOz3JK6bNmJ6BbtK1krcaFFopovnIT47EHahUgahAgXqu30dqE9+PeF/0b1h9/wyfQfLyLFtkZ95LIKWcrrV14Uxby/tC7KN5xJn4Wz9Kdqldoq97AN/k0xmzlm68Q63K/E3bZiiEHs8NNC2nBi6syPkupW3rXt1HKYWhTIAKJEfcSkwb2aWaS7MqogfC9HtLYmIzJ62SrcbYokHD6ayNOHwTVF4zUaGyE/fw+r9ScqkcizVUYdbHzOWOKkaF9XvwSl6TuMYYYrIksewMphSqeKpo4M4qfeRANZCa3Y3Z+SG8PMQ3DrjcsvbiQ9yQRu7QnZVxpJFIxXYMu4iUnGqNfhqa/lxEY8PfN289Lqavtkz/Ic2bj2gB1LtpIr49wVeq04asUaLiEwz/keNJoF8I5ZmB4DGgaw7VgXIM30FsUJ1UbCi8kOYHKP5aKnvTjaL93yr+dZV+CvcIxqE409IdSuYcfVW13OfkSrMUf+Od56ObED+1uef+UsyO9NoSfSSZCyZNQwLTPk+zfkT6YJbkpL0jWnKGoSgqdEtx5phfwxBwLvqwp5rpLavM1un30iPZ3v2Qp6O6i17zPO9DCkOgwTRkRuMQ8MOdPJBSASYApwPZk1rJhsQ4JeJCDAJhsgzsrJ81ByGr3TPDnDkwkabBdC1SCoGlwIJi+snq4DnBZ1YIpGbIr9kmA6Wkek/xwOIJWhEgqwtsIIEJBc9LABW8SqchCYgGWiNxyUjiF/geIzFE72D48BVgt8Anm50ruklQBuvcMA0Z5Ty55pJW77Brktmc2PcP1nkyu3EO6k6UhklBpkZIEprAx2iQsiKVLLsGK5/904G6/wQcSoPo4xf9vDZbwF3IlUlvAYyFSwirSfITWpihD9Mp1zD4mfk2estRB216/79tGa8SK4zDfoLPInfonGWSbBXUAaM3SwTckhB4kijA/O5gjszPCHNmkx76y4dk95ZxTL6AAU0SW7aHT5wo4uMoHo+CGzEtGOTafyMqraq3O7T2O+40gCcWrWIQscXkwI3NP5V1reGEXQBedoC7ltV9hh2By19iDTj+/mjWbO2HeoZcTbOZxP2GVqOszjZSbnDRaBluX7LJcQN7XdOG8fvVn3ItjVyeLBbzU3zO8HmJzyt8flksbMKe+nf99lwAx1fhpmIO2uWr7M7CwbNv8J6nvALNNoJZD1mD1wHyAohwTEJrHr0n/np0GRyugkLAnRP1vdLfGv0xD4qJC8g1/fCrooKadtxceSK20WZr4rCJ57krEn7ZQkX7eeTXeYTqlt9oXlOuYbri9rSAIZo/DK/LoLa/+RcqEkDB)

---

EDIT 2 : the transparent border affect every type of btn, like on:hover, upon clicking for the animation... It seems a more troublesome than usefull idea
Would removing the additionnal width on `outlined` easier and cleaner ?
